### ISSUE TITLE
[Feature Request] Add captions to file list

### ISSUE THREAD

**Issue by Aokromes** [telegramdesktop/tdesktop#3573]
### Steps to reproduce
I have the idea, since api don't allows long filenames to use caption to put the complete filename, i have seen you can search the captions, but when you aren't sure of exact spelling that gives more work, how about show caption text under file date?
(maybe date can be moved next to file size to continue having 3 rows  there)

Something like this: http://i.imgur.com/TAOuee2.png

### Configuration
**Operating system:**
N/R
**Version of Telegram Desktop:**
N/R
**Used theme**:
N/R

**Comment by stek29** [comment]
So you're thinking about showing caption in shared media section?

**Comment by Aokromes** [comment]
yup on that way i can see complete filenames without having to use search.

**Comment by Aokromes** [comment]
added another screenshot to show the reason of this feature request.

**Comment by Aokromes** [comment]
still wanted

**Comment by Aokromes** [comment]
still wanted
